contracts/modules/RevokeModule.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity ^0.8.4;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "interfaces/ISafe.sol";
+import "interfaces/push/INotification.sol";
```

Comment:
maybe move this to `BaseModule` as well?

Reply:
actually might as well move ierc and isafe there as well, no?

Reply:
interesting while trying to settled them back to base module, brownie complains on compiling 😢

Reply:
lets not do this for now